soc/infineon/cat1a/Kconfig

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2024 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Infineon CAT1A devices
+
+# Category definitions
+config SOC_FAMILY_INFINEON_CAT1
+	bool
+
+config SOC_FAMILY_INFINEON_CAT1A
+	bool
+
+# Family definitions
+config SOC_FAMILY_PSOC6
```

Comment:
This is not correct, revert to how it was done previously

Reply:
updated